docs/source/release_notes/release-notes.rst

```diff
@@ -49,6 +49,36 @@ develop
 
     *    - Type
          - Change
+
+    *    - |devbreak| |new|
+         - KVS method ``deleteAllTimestamps`` now also takes a boolean argument specifying if garbage deletion sentinels should also be deleted.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3212>`__)
```

Comment:
This seems a little bit of an unnecessary break, but I realise it's consistent with our TS PRs